packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -144,6 +144,7 @@ const ConditionsForm: React.FC<ConditionsFormProps> = ({
                     id="onsetDate"
                     type="date"
                     name="onsetDate"
+                    max={dayjs(new Date().toUTCString()).format("YYYY-MM-DD")}
```

Comment:
@mherman22  can you add a ticket Id to your title and commit message,https://wiki.openmrs.org/display/docs/Pull+Request+Tips